js/atree.js

```diff
@@ -1280,6 +1250,62 @@ function atree_set_state(node_wrapper, new_state) {
     }
 };
 
+// first key is connector type, second key is highlight, then [x, y] pair of 0-index positions in the tile atlas
+const atreeConnectorAtlasPositions = {
+    "1100": {"0000": [0, 0], "1100": [1, 0]},
+    "1010": {"0000": [2, 0], "1010": [3, 0]},
+    "0110": {"0000": [4, 0], "0110": [5, 0]},
+    "1001": {"0000": [6, 0], "1001": [7, 0]},
+    "0101": {"0000": [8, 0], "0101": [9, 0]},
+    "0011": {"0000": [10, 0], "0011": [11, 0]},
+    "1101": {"0000": [0, 1], "1101": [1, 1], "1100": [2, 1], "1001": [3, 1], "0101": [4, 1]},
+    "0111": {"0000": [5, 1], "0111": [6, 1], "0110": [7, 1], "0101": [8, 1], "0011": [9, 1]},
+    "1110": {"0000": [0, 2], "1110": [1, 2], "1100": [2, 2], "1010": [3, 2], "0110": [4, 2]},
+    "1011": {"0000": [5, 2], "1011": [6, 2], "1010": [7, 2], "1001": [8, 2], "0011": [9, 2]},
+    "1111": {"0000": [0, 3], "1111": [1, 3], "1110": [2, 3], "1101": [3, 3], "1100": [4, 3], "1011": [5, 3], "1010": [6, 3], "1001": [7, 3], "0111": [8, 3], "0110": [9, 3], "0101": [10, 3], "0011": [11, 3]}
+}
+const atreeConnectorAtlasSize = [17, 5]
+// just has the x position, y is based on state
+const atreeNodeAtlasPositions = {
+    "node_0": 0,
+    "node_1": 1,
+    "node_2": 2,
+    "node_3": 3,
+    "node_archer": 4,
+    "node_warrior": 5,
+    "node_mage": 6,
+    "node_assassin": 7,
+    "node_shaman": 8
+}
+const atreeNodeAtlasSize = [17, 5]
```

Comment:
i don't think? the node atlas needs to be 17x5 -- TODO: do some math to compute good sizes

i think it had good (integer/half-integer) %s out of the box with (9, 3) (probably goes down to (8, 2) which is good..? i haven't calculated it)